modules/backup-s3/s3/s3.go

```diff
@@ -90,6 +91,8 @@ func (s *s3) GetObject(ctx context.Context, backupID, key string) ([]byte, error
 		return nil, backup.NewErrInternal(errors.Wrapf(err, "get object '%s'", objectName))
 	}
 
+	monitoring.GetMetrics().BackupRestoreDataTransferred.WithLabelValues("backup-s3").Add(float64(len(contents)))
```

Comment:
`WriteToFile` does not seem to be covered

same applies to other backends

Reply:
That would count the same data twice

Reply:
Fixed